src/routes/blog/post/appwrite-compared-to-supbase/+page.markdoc

```diff
@@ -0,0 +1,165 @@
+
+*Similarities:*
+
+- Appwrite and Supabase provide real-time data synchronization using WebSockets.
+- Both of them support broadcasting changes to large audiences.
+- They both offer client libraries for various programming languages.
+
+*Differences:*
+
+- Supabase supports real-time updates for database changes only, whereas Appwrite also supports updates in other products (e.g., function executions).
+
+## Messaging
+
+Messaging allows you to implement communications in your application. You can use this to send information and updates directly to users through various methods, such as push notifications, SMS, and email. Currently, Supabase has no direct messaging features, whereas Appwrite does.
+
+*Similarities:*
+
+- You can use real-time features along with databases in both Appwrite and Supabase to create a chat application.
+- You can use functions in both Appwrite and Supabase to use external providers for SMS and email.
+
+*Differences:*
+
+- Appwrite Messaging features ten different providers to leverage major communications providers for SMS, email, and push notifications. It offers user segregation, message scheduling, and message previews as well. Supabase offers none of these.
+
+## Support
+
+Any developer-first product, regardless of how good and simple it may be, will need a system for support in case its consumers face any issues. Both Appwrite and Supabase have developed support offerings to ensure their consumers can share feedback and get the necessary help. 
+
+*Similarities*:
+
+- Both Appwrite and Supabase offer comprehensive documentation and community forums for self-help.
+- They both provide paid support plans for priority email assistance.
+- They both have active open-source communities where users can collaborate and share knowledge.
+
+*Differences*:
+
+- The Appwrite community is very active and provides fast and comprehensive support.
+- Supabase provides a higher tier of support for a higher price.
+
+# Conclusion
+
+While Appwrite and Supabase are great Backend-as-a-Service (BaaS) offerings that support numerous SDKs, they differ in engineering approaches. The choice between Appwrite and Supabase hinges on your needs and preferences. 
+
+Here’s a table that compares both Appwrite and Supabase:
+
+| Feature | Appwrite | Supabase |
+| --- | --- | --- |
+| Database | Relational (MariaDB) | Relational (PostgreSQL) |
+| API | Focus on APIs | Supports APIs  |
+| Serverless functions runtimes | 10+ languages supported | Only 1 language supported |
+| Self-hosting / Cloud | Self-hosting and Cloud available | Self-hosting and Cloud available |
+| Supported programming languages | Supports SDKs in 10 languages (JavaScript, Python, PHP, Kotlin, Dart https://appwrite.io/docs/sdks.) | Supports SDKs in 6 languages |
```

Comment:
Was the intention to add a hyperlink here?
![image](https://github.com/appwrite/website/assets/31401437/147a8285-f3eb-4480-b692-6d7a3a242524)

Reply:
i don't know, i copied it from notion, but will get it sorted.

Reply:
This might help: https://github.com/appwrite/website/blob/main/src/routes/blog/post/announcing-appwrite-pro/%2Bpage.markdoc#L28